lib/commons/dom/is-natively-focusable.js

```diff
@@ -31,6 +31,7 @@ function isNativelyFocusable(el) {
     case 'select':
     case 'summary':
     case 'button':
+    case 'iframe':
```

Comment:
I would also like to enable the `cae760` ACT rule (Iframe element has non-empty accessible name). With this fix in, that rule should be consistent.